README.md

```diff
@@ -66,10 +66,8 @@ git clone https://github.com/ethereum/remix-project.git
 * Build `remix-project`:
 ```bash
 cd remix-project
-yarn install
-yarn run build:libs // Build remix libs
```

Comment:
I believe this command is still required to build remix libs

Reply:
@joeizang said to rm it. but if you're sure I'll ut it back.